public/app/features/browse-dashboards/BrowseFolderAlertingPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useMemo } from 'react';
+
+import { Page } from 'app/core/components/Page/Page';
+import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+import { buildNavModel, getAlertingTabID } from 'app/features/folders/state/navModel';
+import { useSelector } from 'app/types';
+
+import { AlertsFolderView } from '../alerting/unified/AlertsFolderView';
+
+import { useGetFolderQuery } from './api/browseDashboardsAPI';
+import { FolderActionsButton } from './components/FolderActionsButton';
+
+export interface OwnProps extends GrafanaRouteComponentProps<{ uid: string }> {}
+
+export function BrowseFolderAlertingPage({ match }: OwnProps) {
+  const { uid: folderUID } = match.params;
+  const { data: folderDTO, isLoading } = useGetFolderQuery(folderUID);
+  const folder = useSelector((state) => state.folder);
+
```

Comment:
Can I be a pain and ask if we can write tests for this?

Something like

```ts
describe("BrowseFolderAlertingPage", () => {
    it("renders correct breadcrumbs"); 
    it("renders folder name");
    it("renders alerts for folder");
    it("shows the alerting tab as selected")
});
```

Likewise for the other one? 🥺